sources/install/package_mobile.sh

```diff
@@ -90,6 +90,16 @@ function install_androguard() {
     add-to-list "androguard,https://github.com/androguard/androguard,Reverse engineering and analysis of Android applications"
 }
 
+function install_mobsf(){
+    colorecho "Installing Mobile Security Framework"
```

Comment:
Please the following comment above `# CODE-CHECK-WHITELIST=add-aliases`
As per https://exegol.readthedocs.io/en/dev/community/contributors.html#function-structure (the doc is still WIP, I'd be happy to have your feedback on this, but that's another topic)

Reply:
done